src/maps/workerFlowRun.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable camelcase */
+import { WorkerFlowRun, WorkerFlowRunResponse, WorkerFlowRuns, WorkerFlowRunsRequest } from '@/models'
+import { MapFunction } from '@/services'
+
+export const mapWorkerFlowRunResponseToWorkerFlowRun: MapFunction<WorkerFlowRunResponse, WorkerFlowRun> = function(source: WorkerFlowRunResponse): WorkerFlowRun {
+  return {
+    workerPoolId: source.worker_pool_id,
+    workerPoolQueueId: source.worker_pool_queue_id,
+    flowRun: this.map('FlowRunResponse', source.flow_run, 'FlowRun'),
+  }
+}
+
+export const mapWorkerFlowRunsToWorkerFlowRunsRequest: MapFunction<WorkerFlowRuns, WorkerFlowRunsRequest> = function(source: WorkerFlowRuns): WorkerFlowRunsRequest {
+  return {
+    worker_pool_queue_names: source.workerPoolQueueNames,
+    scheduled_before: source.scheduledBefore,
+    scheduled_after: source.scheduledAfter,
+    limit: source.limit,
```

Comment:
This allows you to get around eslint:camelcase without an eslint-ignore. Which I think is preferred because at the file level, it could hide other instances that are avoidable. Usually I find most of my maps that deal with pythonic keys put all the python keys as strings. 

```suggestion
    'worker_pool_queue_names': source.workerPoolQueueNames,
    'scheduled_before': source.scheduledBefore,
    'scheduled_after': source.scheduledAfter,
    'limit': source.limit,
```

Reply:
Copy that! Thanks for pointing that out, I saw both options around and was wondering which one is preferable